packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -168,6 +175,18 @@ export default class ListControl extends React.Component {
     onChange((value || List()).push(parsedValue));
   };
 
+  processObjectControlRef = ref => {
+    if (!ref) return;
+    this.validations.push(ref.validateObject);
+    this.listUniqueFieldIds.push(uniq(ref.objectUniqueFieldIds));
+  };
+
+  validateList = () => {
```

Comment:
See first suggestion for `EditorControlPane.js`

```suggestion
  validate = () => {
```